api/exception/handlers.py

```diff
@@ -23,5 +28,9 @@ def global_exception_handler(err):
     descr = str(err)
 
     logger.error(descr, extra=dict(status=code, exception=type(err)))
-    response = {'Code': code, 'Message': 'Something went wrong'}
+    return __handler_response(code)
+
+
+def __handler_response(code, description='Something went wrong'):
```

Comment:
I think you can leverage this also in `boto3_exception_handler`

Reply:
The boto3 exception already has a dict so this `__handler_response` is a way to uniform all errors returned to the nice way boto3 have of doing it